docs/build.md

```diff
@@ -90,13 +95,13 @@ Alternatively a production package similar to what is distributed via [GitHub re
 make prod-package
 ```
 
-Once the packaging is done, assuming the built version is `v0.13.0`, the package is created as `_build/prod/rel/epoch/epoch-0.13.0.tar.gz`.
+Once the packaging is done, assuming the built version is `${VERSION:?}`, the package is created as `_build/prod/rel/epoch/epoch-${VERSION:?}.tar.gz`.
```

Comment:
This sounds a little bit off with variable, what about:

> Once the packaging is done, the package is created in `_build/prod/rel/epoch/` directory, e.g. `_build/prod/rel/epoch/epoch-${VERSION:?}.tar.gz`

Or something similar?